Documentation/Milestone5.MD

```diff
@@ -303,4 +303,45 @@ g)  Team lead ensured that all team members read the final M1 and agree/understa
 
 ## Team Members' Contribution
 
```

Comment:
1) @Brandon: PPT and recording of sections a-c.
@Lionel: Set up Google slides, fill out project information.
PPT and recording of sections f-i.
Documentation Google analytics.
@Luiz: PPT and recording of sections d-e, recording of project demo section c,
Documentation: screenshots.
@Michael: Database Integration
@Rosie: Recording of project demo sections a-b
Documentation: Post-project analysis
@Ruth: Documentation,
Trello Workspace: "Backlog", "In process", and "Done" section
Organization and set up "Team members
Contribution".
Revision history table
Group daily 10 minutes Scrum meetings, or tailored as needed.

2) @Luiz: created WhatsApp group chat
@Luiz: created Jira team's workspace
@Rosie: created GitHub for Team 13 group
@Michael: created Team 13 Home Page at Lamp server
All team members created a LAMP and MySQL account set up, and a home page
All team members completed Atlassian Software Project Management Training
All team members create GitHub team private account
All team members created a Jira Collaboration workspace

3) Members submissions to GitHub team development
     Brandon Basdeo:
     Lionel Munoz:
     Luiz Zullino:
     Michael Hollinger:
     Rosalba Monterrosas:
     Ruth Betancourt: